src/js/views/patients/sidebar/action/action-sidebar-forms_views.js

```diff
@@ -58,7 +58,7 @@ function getSharingOpts(sharing) {
 }
 
 const FormSharingView = View.extend({
-  className: 'sidebar__dialog',
+  className: 'sidebar__dialog u-margin--t-24',
```

Comment:
So that the element has the correct top margin when being shown/hidden on the fly via a web socket notification.